README.md

```diff
@@ -32,6 +32,8 @@ Rector can:
 - [Add typehints based on new types of parent class or interface](/docs/DynamicRectors.md#remove-a-value-object-and-use-simple-type)
 - And much more...
 
+...just look at overview of [all available Rectors](/docs/AllRectorsOverview.md)
```

Comment:
Maybe:
```markdown
All Rectors overview are available in [the documentation](/docs/AllRectorsOverview.md).
```

Reply:
Well, everything is documentation, and link should be clear about where it goes.

I'd exepect "the documention" go to global "/docs".